[2025-09-03T20:04:01.600+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T20:04:01.626+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-03T20:03:00+00:00 [queued]>
[2025-09-03T20:04:01.636+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-03T20:03:00+00:00 [queued]>
[2025-09-03T20:04:01.636+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-03T20:04:01.649+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_foreign_1> on 2025-09-03 20:03:00+00:00
[2025-09-03T20:04:01.657+0000] {standard_task_runner.py:63} INFO - Started process 2830454 to run task
[2025-09-03T20:04:01.662+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'save_foreign_1', 'scheduled__2025-09-03T20:03:00+00:00', '--job-id', '87471', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmpnqpw6vx2']
[2025-09-03T20:04:01.665+0000] {standard_task_runner.py:91} INFO - Job 87471: Subtask save_foreign_1
[2025-09-03T20:04:01.721+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-03T20:03:00+00:00 [running]> on host 1877907bc043
[2025-09-03T20:04:01.880+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='save_foreign_1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-03T20:03:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-03T20:03:00+00:00'
[2025-09-03T20:04:01.882+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T20:04:01.896+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T20:04:02.313+0000] {foreign_1.py:28} INFO - Đã lưu foreign_ALL
[2025-09-03T20:04:02.314+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T20:04:02.882+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HOSE
[2025-09-03T20:04:02.884+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T20:04:03.464+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HNX
[2025-09-03T20:04:03.465+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T20:04:03.915+0000] {foreign_1.py:28} INFO - Đã lưu foreign_UPCOM
[2025-09-03T20:04:03.916+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-03T20:04:03.917+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T20:04:03.931+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=save_foreign_1, run_id=scheduled__2025-09-03T20:03:00+00:00, execution_date=20250903T200300, start_date=20250903T200401, end_date=20250903T200403
[2025-09-03T20:04:03.965+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-03T20:04:04.004+0000] {taskinstance.py:3498} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-09-03T20:04:04.005+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
